packages/odyssey-react-mui/src/labs/AppTile.tsx

```diff
@@ -99,7 +99,8 @@ const ImageContainer = styled("div", {
   maxHeight: APP_TILE_IMAGE_HEIGHT,
   marginBlockEnd: odysseyDesignTokens.Spacing5,
 
-  ["& img"]: {
+  ["img"]: {
+    height: APP_TILE_IMAGE_HEIGHT,
```

Comment:
Is this globally hitting all images? I think we don't want that. Wanna really do a VRT check.

Reply:
No, this only hits images that are inside of this styled parent component.